subprojects/diagnostics/src/integTest/groovy/org/gradle/api/tasks/diagnostics/OutgoingVariantsReportTaskIntegrationTest.groovy

```diff
@@ -39,6 +40,11 @@ class OutgoingVariantsReportTaskIntegrationTest extends AbstractIntegrationSpec
         run ':outgoingVariants'
 
         then:
+        def jarPath = new TestFile(getTestDirectory(), 'build/libs/myLib-1.0.jar').getRelativePathFromBase()
```

Comment:
This can be simplified to just `file(...)`.  This will create the TestFile relative to the test directory for you